scripts/plotting/make_postfit_vgen.py

```diff
@@ -149,25 +151,46 @@ def load_hist(filename, fittype="postfit", helicity=False):
         colors.append("gray")
 
     elif args.helicity_fit is not None:
-        gen = load_hist(args.helicity_fit, "prefit", helicity=True)[{"helicity": -1.0j}]
+        if args.slice_helicity is None:
+            gen = load_hist(args.helicity_fit, "prefit", helicity=False)
 
-        theory_up, theory_down = hist_to_up_down_unc(gen)
+            theory_up, theory_down = hist_to_up_down_unc(gen)
 
-        hists_nom.append(gen)
-        hists_err.extend([theory_up, theory_down])
-        labels.append("Helicity fit prefit")
-        names.append("Helicity fit prefit")
-        colors.append("gray")
+            hists_nom.append(gen)
```

Comment:
Hi, I took this piece of code from Elisabetta to remake a plot, it works fine. But I was thinking that the part that appends to the arrays doesn't need to be duplicated in the two scopes, it's enough to separate the loading of the histogram (unless in the feature you also plan to use different colours or titles accordingly of course)